src/app/layers/qgsapplayerhandling.h

```diff
@@ -157,6 +160,15 @@ class APP_EXPORT QgsAppLayerHandling
     //! Add a list of database layers to the map
     static QList< QgsMapLayer * > addDatabaseLayers( const QStringList &layerPathList, const QString &providerKey, bool &ok );
 
+    enum class DependencyFlag : int
```

Comment:
Can you add the documentation to this enum?

Reply:
Done!